.github/workflows/test-rector.yml

```diff
@@ -68,4 +68,4 @@ jobs:
         run: composer update --ansi --no-interaction
 
       - name: Run static analysis
-        run: vendor/bin/rector process --dry-run --no-progress-bar
```

Comment:
I added the "no-progress-bar" because it doesn't display correctly during actions and ends up adding a lot of debris to the output. If "debug" is mutually exclusive with that option that is fine but I would prefer not to have it.

Reply:
I see, I just want to debug in github action to see what last output that cause hang. I will revert back when everything green again.

Reply:
using latest rector/rector:dev-main seems fixed it, reverted the command.